src/read.c

```diff
@@ -2875,9 +2830,6 @@ Obj Call1ArgsInNewReader(Obj f,Obj a)
 static Int InitKernel (
     StructInitInfo *    module )
 {
-#if !defined(HPCGAP)
-    InitGlobalBag(&ReaderState()->StackNams, "src/read.c:StackNams");
```

Comment:
`StackNams` now is in a `ReaderState` instance on the stack, and hence our stack scanning code ensure it is not garbage collected. So this line can go.